Assets/Scripts/Game/Leveling/AlignmentSystem.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.Events;
+
+namespace RPG
+{
+    public class AlignmentSystem : MonoBehaviour
+    {
+        public UnityEvent<int> onAligneChanged;
+
+        public int Sexiness
+        {
+            get => GameManager.Player.GetProperty<int>(PropertyName.Sexiness);
+            set => GameManager.Player.SetProperty<int>(PropertyName.Sexiness, value);
```

Comment:
This and the other properties should probably be `private set` so that they can't be overwritten outside of this class.